### ISSUE TITLE
[SUGGESTION] | More Events

### ISSUE THREAD

**Issue by SimYRuler** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#749]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
My suggestion is to make more atarting points în history of Azeroth by years so that we get more posibilities:
-Destruction of Well of Eternity
-Rise of Naga
-Death of the LichKing
-Discovery of Pandaria
-The bombing of Theramore
-Invasion of the Legion on Broken Shore
-Rebellion of Nightborne
-End of the Legion
-Battle for Azeroth

This enents  have their own history:
-Destruction of Well of Eternity: The status of the world is clear , Nigh elves are neutral and try to rebuild their empire far from the well influence and going to a druidic road. Humans are starting to build their own keeps , Trolls have become mostly extinct but still around , highborne have an increase depedency for sunwell magic and are in a constant war state with Amani Kingdom. The situation on Northrend is constant , everybody trying to survive.
-Rise of Naga: After the explosion , Queen Azshara struck a deal with N'zoth to save her people , they become naga , starts to explore the seas and make a new empire . Soon she send her armies to besiege the landwalkers. Nobody is safe.
-The bombing of Theramore: Was an act of cowardice from Garrosh Hellscream  to gather the alliance leaders in Theramore and to kill them all . Fortunately they escaped , but this event starts the fire of Horde rebellion
-Discovery of Pandaria: Garrosh Hellscream lust for power made the saving of Pandaria possible . Many wachposts have been build and the continent became a full war zone . Vol'jin rebellion put an end to the war with the Siege of Ogrimmar.
 -Invasion of the Legion on Broken Shore: With the help of Elisande , Legion gain a foothold on Broken Isles .The city of Suramar stood proud before Legion but the deals to save lives and to gain power made the city a place a dispair. The rebellion has started.
-Rebellion of Nightborne: Nightfallen or Nightborne rebels are a group of elves who knew that a way to fight and drive back the Legion must exist. They found a way to tap in the ley lines. 
-End of the Legion: After this events în Suramar , Nightfallen , Alliance and The Horde took the fight to the Legion , bringing them down and starting an age of silence in Broken Isles. 
-Battle for Azeroth: With the Legion gone , and the last act of evil from Sargeras that throw his blade into the core of the planet a new problem occurs .The planet is hurt , a new powerfull mineral can be found , azerite . The goblins found it fast and started to cash in for it . But this being the verry essence of the planet  will make earth unstable. Alliance and The Horde found new allies , build new defences and started a war that seem endless . It is up to everybody to choose what is right for Azeroth. Cleanse of Lordaeron , Burning of Teldrasill , siege of Zandalar and atrack on Brennadam are key points in this chapter .

**Comment by zumbak04** [comment]
> -Destruction of Well of Eternity
> -Rise of Naga

Super doubtful because CK2 is designed to be... not so long.

**Comment by DJScias** [comment]
In case this is considered, a small thing I've noticed (haven't read all dates yet):
```
-Death of the LichKing: After the death of Arthas , the Forsaken become a power in Lordaeron and start to make safer border for them after seeing the human hate for them. And doing so they seek allies in the horde ranks.
```

Arthas dies in WoTLK and the Forsaken are a long-since established by then.

The Forsaken join the Horde in 614 (during TFT, Chronicle Vol 3) and the Lich King meets his demise in.. 619.

**Comment by SimYRuler** [comment]
Yep this post is for dev to think about dates and years :) I only throw an idea :)

**Comment by SimYRuler** [comment]
> > -Destruction of Well of Eternity
> > -Rise of Naga
> 
> Super doubtful because CK2 is designed to be... not so long.

true indeed but a player will play that event to a certain point , i m well aware there is a time limit.

**Comment by zumbak04** [comment]
> true indeed but a player will play that event to a certain point , i m well aware there is a time limit.

It's not because of the limit, it's because of technologies and snowball of lags.. The game started in -10000 year BDP will have 6 tier techs in 600 years.

**Comment by virsia** [comment]
the longest play probably 1000-1500 year before the game kinda freeze due to countless unessential courtiers and character being spawned.
